### PR TITLE
chore(Build): Fix problem in flow types generation

### PR DESCRIPTION
**Pull request by pladaria**
Fixes problem after generating flow defs by moving the overrides logic to the `ts-to-flow` library.

See https://github.com/Telefonica/js-toolbox/pull/8

### CODE REVIEW

File: src/portal.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export const usePortalNodes = (): PortalNodesContext => React.useContext(PortalN
  * This is useful for components that need position fixed, for example
  * for menus, alerts, overlays, etc.
  *
- * @see https://reactjs.org/docs/portals.html
+ * See https://reactjs.org/docs/portals.html
```
Comment (pladaria) [inline src/portal.tsx:50]: for some reason flow-gen was producing a strange output due to this annotation

File: flow-defs/dialog.js.flow [hunk 2]
```diff
@@ -22,7 +22,8 @@ declare type DialogRootState = {
   instanceNumber: number,
 };
 declare export default class DialogRoot
-  mixins React.Component<DialogRootProps, DialogRootState> {
+  mixins React.Component<DialogRootProps, DialogRootState>
+{
```
Comment (atabel) [inline flow-defs/dialog.js.flow:26]: weird formating

Comment (pladaria) [reply flow-defs/dialog.js.flow:26]: this is autogenerated :shrug:

Comment (atabel) [reply flow-defs/dialog.js.flow:26]: do you know why it changed?

**Review by atabel — APPROVED** [review]

File: flow-defs/dialog.js.flow (hunk 2 above)
Comment (pladaria) [reply flow-defs/dialog.js.flow:26]: I guess because prettier was updated

**Review by anabelengp — APPROVED** [review]

**Pull request merged**